src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1742,14 +1742,21 @@ await ServiceHostInstance.SendEvent(
                                    new NonTSqlParams
                                    {
                                        OwnerUri = uri,
-                                       NonTSqlKeyword = null
+                                    NonTSqlKeyword = null,
                                    });
-
                 return true;
             }
+
+            HashSet<string> identifiers = new HashSet<string>();
+            IList<SqlIdentifier> scriptIdentifiers = parseResult.Script.RetrieveAllIdentifiers();
+            foreach (var identifier in scriptIdentifiers)
```

Comment:
Generally nice for readability to avoid `var` unless it's apparently from the same line what type it is.